### DISCUSSION TITLE
icon color when off and reduce space

### DISCUSSION THREAD

**Discussion by R3Cardoso** [piitaya/lovelace-mushroom#1157] [Q&A]
Hello, I have this code:
```
      - type: custom:mushroom-light-card
        entity: light.teto_quartoprincipal
        show_color_temp_control: true
        show_brightness_control: true
        collapsible_controls: false
        layout: horizontal
        fill_container: false
        use_light_color: true
```

How can I change the icon color when the light is off?

And about this space between light and brightness, how can I reduce it? Making for exemple the brightness bar bigger.
<img width="490" alt="image" src="https://github.com/piitaya/lovelace-mushroom/assets/119068070/c7dce9bd-417c-4cb0-80ff-dab90576fda0">

**Comment by R3Cardoso** [comment]
icons colors resolved :)
space not yet, but solve another solution.

Anyway, it's possible to make the change I request on that space.

**Reply by Timmiej93** [reply]
If you solve your own issue, please share how you managed to solve it. People with the same issue you had are finding this page, only to discover that you solved the problem, but they still don't know how to solve it.

**Reply by R3Cardoso** [reply]
in fact I found just a couple or hours ago :D

it's just this style:

>         card_mod:
>           style: |
>             mushroom-state-item {
>               margin-right: -80px !important;
>               margin-left: -30px  !important;
>               margin-top: -30px  !important;
>               margin-bottom: -30px  !important;
>             }

What removes  the space is the margin-right. it's not the perfect solution, but it worked for me.
